satpy/composites/aux_data.py

```diff
@@ -0,0 +1,164 @@
+# Copyright (c) 2015-2025 Satpy developers
+#
+# This file is part of satpy.
+#
+# satpy is free software: you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later
+# version.
+#
+# satpy is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR
+# A PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along with
+# satpy.  If not, see <http://www.gnu.org/licenses/>.
+
+"""Compositors using auxiliary data."""
+
+from __future__ import annotations
+
+import logging
+import os
+
+import satpy
+from satpy.aux_download import DataDownloadMixin
+
+from .core import GenericCompositor
+
+LOG = logging.getLogger(__name__)
+
+
+class StaticImageCompositor(GenericCompositor, DataDownloadMixin):
```

Comment:
I'm not sure how I feel about `aux_data` for this module name for this compositor. Yes it takes outside data ("aux" data), but 99% of the time this is to provide a background or mask. I...ugh I don't know.